README.md

```diff
@@ -504,18 +588,22 @@ We're glad you asked! Please visit our [CONTRIBUTING](CONTRIBUTING.md) doc for d
+
+### Is [Pulumi Cloud](https://app.pulumi.com/?utm_campaign=pulumi-examples-github-repo&utm_source=github.com&utm_medium=examples-faq) free?
+
+Individual accounts are always free and allow unlimited resources. See our [pricing page](https://www.pulumi.com/pricing/?utm_campaign=pulumi-examples-github-repo&utm_source=github.com&utm_medium=examples-faq) for other options.
 
-### Is [Pulumi Cloud](https://app.pulumi.com/) free?
+## Pulumi roadmap
 
-Individual accounts are always free and allow unlimited resources. See our [pricing page](https://www.pulumi.com/pricing/) for other options.
+Review the planned work for the upcoming quarter and a selected backlog of issues that are on our mind but not yet scheduled on the [Pulumi Roadmap.](https://github.com/orgs/pulumi/projects/44)
 
-## Further Reading
+## Pulumi resources
 
 Delve deeper into our project with additional resources:
 
-- [Get Started with Pulumi](https://www.pulumi.com/docs/get-started/): Deploy a simple application in AWS, Azure, Google Cloud, or Kubernetes using Pulumi.
-- [Registry](https://www.pulumi.com/registry/): Search for packages and learn about the supported resources you need. Install the package directly into your project, browse the API documentation, and start building.
-- [Try Pulumi AI](https://www.pulumi.com/ai) - Use natural-language prompts to generate Pulumi infrastructure-as-code programs in any language.
-- [Documentation](https://www.pulumi.com/docs/): Learn about Pulumi concepts, follow user guides, and consult the reference documentation.
-- [Pulumi Blog](https://www.pulumi.com/blog/) - Stay in the loop with our latest tech announcements, insightful articles, and updates.
+- [Get Started with Pulumi](https://www.pulumi.com/docs/get-started/?utm_campaign=pulumi-examples-github-repo&utm_source=github.com&utm_medium=examples-resources): Deploy a simple application in AWS, Azure, Google Cloud, or Kubernetes using Pulumi.
+- [Registry](https://www.pulumi.com/registry/?utm_campaign=pulumi-examples-github-repo&utm_source=github.com&utm_medium=examples-resources): Search for packages and learn about the supported resources you need. Install the package directly into your project, browse the API documentation, and start building.
+- [Try Pulumi AI](https://www.pulumi.com/ai/?utm_campaign=pulumi-examples-github-repo&utm_source=github.com&utm_medium=examples-resources) - Use natural-language prompts to generate Pulumi infrastructure-as-code programs in any language.
+- [Documentation](https://www.pulumi.com/docs/?utm_campaign=pulumi-examples-github-repo&utm_source=github.com&utm_medium=examples-resources): Learn about Pulumi concepts, follow user guides, and consult the reference documentation.
+- [Pulumi Blog](https://www.pulumi.com/blog/?utm_campaign=pulumi-examples-github-repo&utm_source=github.com&utm_medium=examples-resources) - Stay in the loop with our latest tech announcements, insightful articles, and updates.
```

Comment:
Bit of a nit, but I'd probably reorder these by likely relevance:

* Get started
* Docs
* Blog
* Registry
* AI